components/brave_ads/core/internal/ads/serving/choose/eligible_ads_predictor_util.h

```diff
@@ -159,6 +164,43 @@ CreativeAdPredictorMap<T> ComputePredictorFeaturesAndScores(
   return creative_ad_predictors_with_features;
 }
 
+template <typename T>
+std::vector<int> ComputeVoteRegistry(
+    const std::vector<T>& creative_ads,
+    const TextEmbeddingHtmlEventList& text_embedding_html_events) {
+  DCHECK(!creative_ads.empty());
+
+  std::vector<int> vote_registry(creative_ads.size(), 0);
```

Comment:
nit: we can remove the `0` and just have `std::vector<int> vote_registry(creative_ads.size());`